package.json

```diff
@@ -16,20 +16,22 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@babel/helper-plugin-utils": "^7.0.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.4.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@babel/plugin-transform-classes": "^7.4.4",
-    "@babel/plugin-transform-exponentiation-operator": "^7.2.0",
-    "@babel/plugin-transform-jscript": "^7.2.0",
-    "@babel/plugin-transform-member-expression-literals": "^7.2.0",
-    "@babel/plugin-transform-property-literals": "^7.2.0",
-    "@babel/plugin-transform-property-mutators": "^7.2.0",
-    "@babel/plugin-transform-runtime": "^7.4.4",
-    "@babel/plugin-transform-template-literals": "^7.4.4",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/helper-plugin-utils": "^7.8.3",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.8.3",
+    "@babel/plugin-proposal-numeric-separator": "^7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "^7.9.0",
+    "@babel/plugin-proposal-optional-catch-binding": "^7.8.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.9.0",
+    "@babel/plugin-transform-classes": "^7.9.2",
+    "@babel/plugin-transform-exponentiation-operator": "^7.8.3",
+    "@babel/plugin-transform-jscript": "^7.8.3",
```

Comment:
I think we should consider removing this one. It's trying to work around IE8 bugs, but it super bloats the code: https://kangax.github.io/nfe/#jscript-bugs

I doubt Chakra has any those bugs (also I'm pretty sure we disable this internally)

Reply:
Im cool with putting this + browser targets in another PR so we can group a breaking change.